deel-3-webpagina/styles/styles.css

```diff
@@ -4,11 +4,33 @@ body {
     text-align: center;
     margin: 0 auto;
     background-color: #F2F3D9;
+    box-sizing: border-box;
 
     display: flex;
     flex-direction: column;
 }
 
+nav {
+    height: 8%;
+    background-color: #030027;
+
+    color: white;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+}
+
+.nav-row-1 {
+    justify-content: space-around;
+}
+
+.nav-row-2 {
```

Comment:
you could have put these row properties under one class so you didn't have to repeat them 3 times